package.json

```diff
@@ -80,7 +80,9 @@
 		"@wordpress/url": "file:packages/url",
 		"@wordpress/viewport": "file:packages/viewport",
 		"@wordpress/warning": "file:packages/warning",
-		"@wordpress/wordcount": "file:packages/wordcount"
+		"@wordpress/wordcount": "file:packages/wordcount",
+		"fbjs": "3.0.0",
+		"react-native-video": "git+https://github.com/wordpress-mobile/react-native-video.git#97792c7737b234fe48840d443a9b3135711e5df7"
```

Comment:
I think these should be in `devDependencies` and could be even better to have them in `react-native-editor/package.json` if possible.